docs/content/en/docs/troubleshooting/troubleshooting.md

```diff
@@ -248,6 +248,40 @@ kubectl logs <podname> -n <namespace> --kubeconfig=<kubeconfig>
 ....
 ```
 
+## Kubectl commands return dial tcp: i/o timeout
+
+If you are unable to run kubectl commands on a cluster due to timeout errors, then check if the server endpoint in the kubeconfig matches the control plane's endpoint in the infrastructure provider. If the endpoints do not match, you can ssh into the control plane node to gather logs instead. The kubelet logs can be obtained by running `journalctl -u kubelet.service --no-pager`. It may also be helpful to look at kube-vip logs, which can be found in the `/var/log/pods/kube-system_kube-vip-*` directory. 
```

Comment:
why would server endpoint not matching cp endpoint?
timeout can be caused by many issues -- mostly the client can't interact with API server

also does the `journalctl` cmd works for both BR and ubuntu?